modules/test/playwright/tests/style-book-web/previewStyleBookOnDisplayPageTemplates.spec.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, expect, mergeTests} from '@playwright/test';
+
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {pageEditorPagesTest} from '../../fixtures/pageEditorPagesTest';
+import {styleBookPageTest} from '../../fixtures/styleBookPageTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import {PageEditorPage} from '../../pages/layout-content-page-editor-web/PageEditorPage';
+import {DisplayPageTemplatesPage} from '../../pages/layout-page-template-admin-web/DisplayPageTemplatesPage';
+import {StyleBooksPage} from '../../pages/style-book-web/StyleBooksPage';
+import getRandomString from '../../utils/getRandomString';
+import {blogsPagesTest} from '../blogs-web/fixtures/blogsPagesTest';
+
+const test = mergeTests(
+	blogsPagesTest,
+	isolatedSiteTest,
+	loginTest(),
+	pageEditorPagesTest,
+	styleBookPageTest,
+	workflowPagesTest
+);
+
+const changeStyleAndCheckPreview = async (
+	colorHEX: string,
+	colorRGB: string,
+	displayPageTemplatesPage: DisplayPageTemplatesPage,
+	page: Page,
+	pageEditorPage: PageEditorPage,
+	site: Site,
+	styleBooksPage: StyleBooksPage,
+	displayPageTemplateName: string
+) => {
+	await styleBooksPage.goto(site.friendlyUrlPath);
+	await styleBooksPage.editStyleBook('Test Style Book Name');
```

Comment:
Since 'Test Style Book Name' is used multiple times, it should be a variable that is reused.

Reply:
Also it would be better to be able to use the randomString generator for the name instead of a static string. This will need to be generated in the test and passed to this method.